src/styles/antd-overrides/tab.scss

```diff
@@ -11,10 +15,15 @@
   color: var(--text-brand-primary);
 }
 
-.ant-tabs-ink-bar {
-  background: var(--text-brand-primary);
+.ant-tabs-tab.ant-tabs-tab-active{
+  font-weight:700;
 }
 
-.ant-tabs {
+ .ant-tabs-tab-btn, .ant-tabs-tab .ant-tabs-tab-btn {
```

Comment:
**Question:** is the first rule here `.ant-tabs-tab-btn` necessary?

Reply:
great catch!